src/index.ts

```diff
@@ -1,26 +1,35 @@
 function debounce<T, A extends Array<unknown>>(fn: (...args: A) => Promise<T>, ms = 0) {
```

Comment:
could you please change the indent here to be spaces, size 2 just like it was before?
maybe you could just add `.editorconfig` file in the root folder:

```
root = true
[*]
indent_style = space
indent_size = 2
end_of_line = true
insert_final_newline = true
trim_trailing_whitespace = true
```